src/DynamoCore/Configuration/PreferenceSettings.cs

```diff
@@ -613,6 +613,8 @@ public string PythonTemplateFilePath
         /// </summary>
         public bool EnableNodeAutoComplete { get; set; }
 
+        public bool DefaultEnableLegacyPolyCurveBehavior { get; set; }
```

Comment:
/// summary please

Reply:
Is it useful for this config name to be prefixed with “Default”?

Reply:
Yes, there are 2 flags - one at the preferences level, and the second, at the workspace level. If there is no workspace level setting, the preference setting is chosen by default, whereas if the workspace setting does exist, it overrides the preference setting.